### ISSUE TITLE
asana.error.InvalidRequestError: Invalid Request: Missing required `team` field

### ISSUE THREAD

**Issue by DanRya** [ishalyminov/wunderlist2asana#7]
Hi I'm getting this strange error while trying to import my Wunderlist tasks into Asana. Any ideas?

`(base) C:\Users\WDAGUtilityAccount\wunderlist2asana-master>python wunderlist_to_asana.py Tasks.json <removed> <removed>`
`C:\ProgramData\Miniconda3\lib\site-packages\asana\client.py:153: UserWarning: This request is affected by the "string_ids" deprecation. Please visit this url for more info: https://asa.na/api-string-ids`
`Adding "string_ids" to your "Asana-Enable" or "Asana-Disable" header will opt in/out to this deprecation and suppress this warning.`
 ` warnings.warn(message)`
`Processing Wunderlist list "inbox" (1/7)`
`Traceback (most recent call last):`
`File "wunderlist_to_asana.py", line 121, in <module>
    move_content(args.wunderlist_backup, args.asana_token, args.workspace_name)`
`File "wunderlist_to_asana.py", line 46, in move_content
    {'name': project['title']}`
  `File "C:\ProgramData\Miniconda3\lib\site-packages\asana\resources\gen\projects.py", line 51, in create_in_workspace
    return self.client.post(path, params, **options)`
 `File "C:\ProgramData\Miniconda3\lib\site-packages\asana\client.py", line 200, in post
    return self.request('post', path, data=body, headers=headers, **options)`
 `File "C:\ProgramData\Miniconda3\lib\site-packages\asana\client.py", line 89, in request
    raise STATUS_MAP[response.status_code](response)`
`asana.error.InvalidRequestError: Invalid Request: Missing required `team` field`

**Comment by ishalyminov** [comment]
Hi,

Thanks for reporting this!
Interesting, this 'team' field was never required in my experiments - are you using some special type of project/workspace in Asana?

**Comment by DanRya** [comment]
Thank you for replying so quickly!

I don't believe so but I wasn't the one in my organisation to set this up.
I will ask my manager next week if he can give me any more information.

Thank you for making this useful script and hopefully together we can get
it working for our setup! I expect there will be many more people using the
script soon with Wunderlist shutting down in May.

**Comment by ishalyminov** [comment]
Wow, didn't know about Wunderlist shutting down , thanks for the insight:)
Sure thing, I'll be happy to help make this work for you!

**Comment by SilverLightSaber** [comment]
I ran into this today, I found the issue and where i needed to make the modification.  I haven't programed in Python in over a decade and I just became aware of asana.  So my technical knowledge is very limited, and I probably can't answer any questions on why it works in some cases and not others.

The comments for create_in_workspace in projects.py states when team will be required.  It is also referenced in many other locations in that file.

> def create_in_workspace(self, workspace, params={}, **options):
>         """If the workspace for your project _is_ an organization, you must also
>         supply a `team` to share the project with.
> 
>         Returns the full record of the newly created project.
> 
>         Parameters
>         ----------
>         workspace : {Gid} The workspace or organization to create the project in.
>         [data] : {Object} Data for the request
>         """
>         path = "/workspaces/%s/projects" % (workspace)
>         return self.client.post(path, params, **options)

After those changes I was able to do an import.  I'm not sure if this would need to be done elsewhere since I stopped after my first test import.

Basically the team (when required) needs to have the ID (represented by the ones below)  as a parameter.  You can get the ID in the URL when you select that team. Ex https://app.asana.com/0/1111111111111111/overview and insert it into line 46, I just entered a bunch of zeroes instead of the actual ID

`{'name': project['title'],'team':'1111111111111111'}`

**Comment by ishalyminov** [comment]
Thanks for looking into this @SilverLightSaber! I've applied some changes, and organizations are properly supported now.

**Comment by DanRya** [comment]
You're all so amazing. Thank you so much for fixing this and getting it working perfectly for us. This is going to save us SO much time and probably many more people with the shutdown  of Wunderlist.
